x/deployment/keeper/keeper.go

```diff
@@ -219,4 +262,13 @@ func (k Keeper) updateGroup(ctx sdk.Context, group types.Group) {
 	store := ctx.KVStore(k.skey)
 	key := groupKey(group.ID())
 	store.Set(key, k.cdc.MustMarshalBinaryBare(group))
+
+	// Update the Open Groups prefixed index
+	openKey := groupOpenKey(group.ID())
+	switch group.State {
+	case types.GroupOpen:
+		store.Set(openKey, []byte{})
+	default:
+		store.Delete(openKey)
+	}
```

Comment:
only substantive comment is what I mentioned in the call: put this in a function somewhere & have it be the only place that the index is modified.

Reply:
I've collocated all of the modifications to the OpenGroups index into this method `updateOpenGroupsIndex()`. Both creation and deletion are all handled here and used by callers.